src/libraries/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -57,7 +57,7 @@ public static void WindowWidth_SetInvalid_ThrowsArgumentOutOfRangeException(int
         }
         else
         {
-            AssertExtensions.Throws<ArgumentOutOfRangeException>("width", () => Console.WindowWidth = value);
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("value", () => Console.WindowWidth = value);
```

Comment:
The test suite was flagging these as failures, so I updated the test.

Reply:
Thank you for doing that! 👍 I initially blamed #79460 for that, then checked the history and realized that it was #79460.

@stephentoub Is it better to revert to the old behavior (use names "width" and "height") or stick with "value"? The change was introduced in .NET 8 Preview 1.

Reply:
Historically I believe we settled on `value` for the param name in argument exceptions from setters. (Personally I preferred something suggestive of the property name as it gives more useful diagnostics if you only have the message.)

Incidentally I believe that we don't consider changes to these to be breaking.

Reply:
I've adopted the more descriptive names.